problems/removeEvenStrings.js

```diff
@@ -19,8 +19,21 @@
 * returns: []
 */
 
-function removeEvenStrings() {
+function removeEvenStrings(arr) {
 
+    //     let newArray = arr.filter((arr) => {
+    //         return arr % 2 === 0
+    //     })
+    //     return newArray
+    // };
+
+    let even = ['']
```

Comment:
Do you want even to be an array with an empty string in it or just an empty array?